src/main/java/uk/gov/hmcts/divorce/common/event/CreateTestCase.java

```diff
@@ -69,7 +69,36 @@ public void configure(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
             .grant(CREATE_READ_UPDATE, roles.toArray(UserRole[]::new))
             .grantHistoryOnly(SUPER_USER, CASE_WORKER, LEGAL_ADVISOR, SOLICITOR, CITIZEN))
             .page("All Structured Data Test Page", this::midEvent)
-                .done();
+
+
+            .complex(CaseData::getCicCase)
+               .label("caseObject", "CIC  Case Categorisation \r\n" + "\r\nCase Record for [DRAFT]")
+              .mandatoryWithLabel(CicCase::getCaseCategory, "")
+              .mandatoryWithLabel(CicCase::getCaseSubcategory, "CIC Case Subcategory")
+            .optionalWithLabel(CicCase::getComment,"Comments")
+            .done()
+            .page("Date")
+            .label("dateObject","when was the case Received?\r\n" + "\r\nCase Record for [DRAFT]\r\n" + "\r\nDate of receipt")
+            .complex(CaseData::getCicCase)
+               .mandatoryWithLabel(CicCase::getCaseReceivedDate, "")
+            .done()
+            .page("objectSubject")
+            .label("subjectObject", "Which parties are named on the tribunal form?\r\n" + "\r\nCase record for [DRAFT]")
+            .complex(CaseData::getCicCase)
+            .mandatoryWithLabel(CicCase::getSubjectCIC,"Select all that apply.")
+            .optional(CicCase::getApplicantCIC,"")
+            .optional(CicCase::getRepresentativeCic,"")
+            .done()
+            .page("applicantDetailsObject")
+            .label("applicantDetailsObject","Who is the subject of this case?\r\n" + "\r\nCase record for [DRAFT]")
+            .complex(CaseData::getCicCase)
+            .mandatory(CicCase::getFullName)
+            .optional(CicCase::getAddress)
+            .optional(CicCase::getPhoneNumber)
+            .optional(CicCase::getEmail)
+            .mandatoryWithLabel(CicCase::getDateOfBirth,"")
+            .mandatoryWithLabel(CicCase::getContactDetailsPrefrence,"")
+            .done();
```

Comment:
This is very long and poses some readability issues, is it possible to chunk it into logical portions, and make it a little more readable?